assets/js/scripts.js

```diff
@@ -1153,14 +1253,24 @@ document.getElementById('open-custom-marker-color-modal').addEventListener('clic
   });
 });
 
+if (isMobile()) {
+  searchHotkey.style.display = 'none';
+
+  Settings.overrideBrowserSearch = false;
+  overrideSearch.check = false;
+  overrideSearch.disabled = true;
+  overrideSearch.parentElement.parentElement.classList.add('disabled');
+  overrideSearch.parentElement.parentElement.disabled = true;
+}
+
 function filterMapMarkers() {
   uniqueSearchMarkers = [];
   let filterType = () => true;
   let enableMainCategory = true;
 
   if (Settings.filterType === 'none') {
-    if (document.getElementById('search').value)
-      MapBase.onSearch(document.getElementById('search').value);
+    if (searchInput.value)
+      MapBase.onSearch(searchInput.value, true);
     else
       uniqueSearchMarkers = MapBase.markers;
 

```

Comment:
My suggestion is to use parameter to prevent infinite mutual recursion MapBase.onSearch and filterMapMarkers

```javascript


function filterMapMarkers({ useItemsQuery }) {
  uniqueSearchMarkers = [];
  let filterType = () => true;
  let enableMainCategory = true;

  if (Settings.filterType === 'none') {
    if (searchInput.value && useItemsQuery)
      MapBase.onSearch(searchInput.value, true);
    else
      uniqueSearchMarkers = MapBase.markers;

    MapBase.addMarkers();
    return;
  }
...
```

Reply:
Maybe give that parameter a better name like `useSearchInput`. That will be more descriptive